llvm/include/llvm/TargetParser/AArch64TargetParser.h

```diff
@@ -343,7 +350,9 @@ bool isX18ReservedByDefault(const Triple &TT);
 // themselves, they are sequential (0, 1, 2, 3, ...).
 uint64_t getCpuSupportsMask(ArrayRef<StringRef> FeatureStrs);
 
-void PrintSupportedExtensions(StringMap<StringRef> DescMap);
+void PrintSupportedExtensions();
+
+void printEnabledExtensions(std::set<StringRef> EnabledFeatureNames);
```

Comment:
Can this be `std::set<StringRef>&` to avoid a (maybe large) copy?

Reply:
Might as well be const correct too if you're changing it.

Reply:
Thanks for catching this! Not sure how I missed it 🤦